src/main/java/com/hivemq/bootstrap/netty/initializer/AbstractChannelInitializer.java

```diff
@@ -73,45 +71,31 @@ protected void initChannel(@NotNull final Channel ch) throws Exception {
         ch.pipeline().addLast(MQTT_MESSAGE_DECODER, new MQTTMessageDecoder(channelDependencies));
```

Comment:
FIRST_ABSTRACT_HANDLER is not needed and should be removed
GLOBAL_THROTTLING_HANDLER should only be added when throttling is enabled (getWriteLimit > 0 || getReadLimit > 0)

Reply:
FIRST_ABSTRACT is currently needed, but i can remove that